server/data/restClient.ts

```diff
@@ -29,6 +29,9 @@ interface PostRequest {
 
 export type RestClientError = HttpError
 
+const readRetryCount = 1
+const writeRetryCount = 1
```

Comment:
`1` _is_ the default for superagent, according to its docs -- so we could just remove the lines, but I kinda feel it's worth being explicit